deployments/ansible/roles/collector/tasks/main.yml

```diff
@@ -1,13 +1,29 @@
 ---
+- name: Define acceptable distros For splunk-otel-collector
+  set_fact:
+    debian_distro: ['Debian', 'Ubuntu']
+    rhel_distro: ['CentOS', 'RedHat', 'Red Hat Enterprise Linux', 'Amazon']
```

Comment:
I don't see these facts being used. Doesn't seem like this task is needed at all

Reply:
Ok I will remove it.